pkg/framework/aries/framework.go

```diff
@@ -251,6 +251,12 @@ func (a *Aries) Context() (*context.Provider, error) {
 	)
 }
 
+// Messenger returns messenger for sending messages through this agent framework
+// TODO should use dedicated messenger interface instead of Outbound dispatcher [Issue #1058]
+func (a *Aries) Messenger() dispatcher.Outbound {
+	return a.outboundDispatcher
+}
+
```

Comment:
If I heard correctly - the messenger should not be exposed through the framework. It should be an internal implementation detail (for protocol services to communicate with each other).

Reply:
No, this is for external code to send message through already running agent without using context

Reply:
We should see if there is a better idea as a followup.